src/main/java/frc/robot/subsystems/TankDriveSubsystem.java

```diff
@@ -26,6 +27,8 @@ public class TankDriveSubsystem extends SubsystemBase {
     private CANSparkMax secondaryRightMotor;
     private CANSparkMax primaryLeftMotor;
     private CANSparkMax secondaryLeftMotor;
+    private double speedCoefficient = 1;
```

Comment:
This value is never used.

Where should it makes its way to? Perhaps `pid.setOutputRange` ?

Reply:
SpeedCoefficient should be the coefficient of input.
So we could put it in outputrange, which means that PID is restricted from going to that output.
However, it likely needs to be in setReference as well because that is when it is recieving the controller input. I think that setting it to the outputrange might mess a bit with the oscillatory nature of the PID though? I think we'd have to test to see whether or not it would have a significant/insignificant effect.

Reply:
Something to note is that this is the new_intake branch, but for the DriveTrain, I don't believe the controls have been changed to the POV on the controller (four arrows/cross-shape ) yet.

Reply:
At this point, new_intake is where everything is